### PR TITLE
Add XML Web Services to 21.0.0.2-beta blog

### PR DESCRIPTION
**Pull request by austin0**
Adds the Jakarta XML Web Services 3.0 write up to the 21.0.0.1-beta blog.

https://github.com/OpenLiberty/open-liberty/issues/15689

https://blogs-draft-openlibertyio.mybluemix.net/blog/2021/01/26/ee9-messaging-security-21002-beta.html

### CODE REVIEW

File: posts/2021-01-26-ee9-messaging-security-21002-beta.adoc [hunk 1]
```diff
@@ -116,6 +116,35 @@ The following features can be included in your server.xml:
 
 Additional information about the Jakarta Security 2.0 specification can be found link:https://jakarta.ee/specifications/security/2.0/[here].
 
+[#xmlws]
+=== Jakarta XML Web Services 3.0
+
+Jakarta XML Web Services 3.0 is an API that enables applications to create SOAP based Web Services. Support provides the following updated APIs: XML Web Services 3.0, Jakarta Web Services Metadata 3.0, Jakarta SOAP with Attachments 2.0. Support also introduces the `xmlWS-3.0` feature, as well as the Application Client Container support for XML Web Services 3.0 via the `jakartaeeClient-9.0` convenience feature.
```
Comment (tevans78) [inline posts/2021-01-26-ee9-messaging-security-21002-beta.adoc]:
```suggestion
Jakarta XML Web Services 3.0 is an API that enables applications to create SOAP based Web Services. The `xmlWS-3.0` feature provides API updates to the Jakarta XML Web Services 3.0 API, Jakarta Web Services Metadata 3.0 API and Jakarta SOAP with Attachments 2.0 API. The `jakartaeeClient-9.0` feature has been also been updated to provide Application Client Container support for the Jakarta XML Web Services 3.0 API.
```

File: posts/2021-01-26-ee9-messaging-security-21002-beta.adoc [hunk 2]
```diff
@@ -116,6 +116,35 @@ The following features can be included in your server.xml:
...
+
+The `xmlWS-3.0` feature provides the Jakarta EE 9 XML Web Services 3.0 tools for Top Down and Bottom Up Web Services Creation: `wsgen` and `wsimport`. These can be found in a new location in the Open Liberty image: wlp/bin/xmlWS/. Both commands preform nearly identical to the Jakarta EE 8 tooling, with the exception that the `-target` argument is no longer required.
+
+The implementation of Jakarta XML Web Services 3.0 in Open Liberty is based on Apache CXF 3.4.1. This provides numerous improvements and fixes over the forked version of CXF 2.6.2 in use by the Jakarta EE 8 `jaxws-2.3` feature (Note: Not all CXF 3.4.1 functionality is available in `xmlWS-3.0`, but watch this space for updates.)
```
Comment (tevans78) [inline posts/2021-01-26-ee9-messaging-security-21002-beta.adoc]: I think maybe this paragraph should be removed... I don't like advertising a specific implementation version or hinting at support for future function.

File: posts/2021-01-26-ee9-messaging-security-21002-beta.adoc [hunk 3]
```diff
@@ -116,6 +116,35 @@ The following features can be included in your server.xml:
...
+To run Jakarta XML Web Services on the Application Client Container: Here's an example of what the client.xml configuration looks:
+
+[source, xml]
+----
+ <featureManager>
+       <feature>jakartaeeClient-9.0</feature>
+ </featureManager>
+----
+
+For more information see the link:https://jakarta.ee/specifications/xml-web-services/[Jakarta XML Web Services Specification], and the link:https://cxf.apache.org/docs/34-migration-guide.html[Apache CXF 3.4 Migration Guide].
```
Comment (tevans78) [inline posts/2021-01-26-ee9-messaging-security-21002-beta.adoc]: Link directly to the 3.0 version: https://jakarta.ee/specifications/xml-web-services/3.0/

Comment (tevans78) [reply posts/2021-01-26-ee9-messaging-security-21002-beta.adoc]: I think the Apache CXF Migration Guide link should be removed

File: posts/2021-01-26-ee9-messaging-security-21002-beta.adoc (hunk 2 above)
Comment (austin0) [reply posts/2021-01-26-ee9-messaging-security-21002-beta.adoc]: Removed paragraph and changed `preform` -> `perform`

File: posts/2021-01-26-ee9-messaging-security-21002-beta.adoc (hunk 1 above)
Comment (austin0) [reply posts/2021-01-26-ee9-messaging-security-21002-beta.adoc]: Updated paragraph.

File: posts/2021-01-26-ee9-messaging-security-21002-beta.adoc (hunk 3 above)
Comment (austin0) [reply posts/2021-01-26-ee9-messaging-security-21002-beta.adoc]: Updated link and remove link to migration guide.

**Review by tevans78 — APPROVED** [review]
LGTM

**Pull request merged**